### PR TITLE
PMM-7671 - Fix failing

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

**Comment by IvanaBrecherova** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/8199/tests/ Fixed

File: tests/verifyAWSRDSPostgreSQLInstance_test.js [hunk 1]
```diff
@@ -29,12 +29,28 @@ xScenario(
     pmmInventoryPage.verifyRemoteServiceIsDisplayed(serviceName);
     await pmmInventoryPage.verifyAgentHasStatusRunning(serviceName);
     await pmmInventoryPage.verifyMetricsFlags(serviceName);
-    // Wait until data in QAN and dashboard with slowlog will be loaded.
-    I.wait(30);
+  },
+);
+
+Scenario(
+  'PMM-T716 - Verify adding PostgreSQL RDS monitoring to PMM via UI DASHBOARD @instances',
```
Comment (puneet0191) [inline tests/verifyAWSRDSPostgreSQLInstance_test.js:36]: I think you meant verify Dashboard for Postgres RDS added via UI, right?

File: tests/verifyAWSRDSPostgreSQLInstance_test.js [hunk 2]
```diff
@@ -29,12 +29,28 @@ xScenario(
...
+
     I.amOnPage(dashboardPage.postgresqlInstanceOverviewDashboard.url);
     dashboardPage.applyFilter('Node Name', serviceName);
     await dashboardPage.verifyThereAreNoGraphsWithNA();
     await dashboardPage.verifyThereAreNoGraphsWithoutData();
+  },
+).retry(2);
+
+Scenario(
+  'PMM-T716 - Verify adding PostgreSQL RDS monitoring to PMM via UI QAN @instances',
```
Comment (puneet0191) [inline tests/verifyAWSRDSPostgreSQLInstance_test.js:50]: The same here, I think you meant verify QAN for Postgres RDS added via UI

Comment (puneet0191) [reply tests/verifyAWSRDSPostgreSQLInstance_test.js:50]: Because it is not really making sense adding Postgres via UI QAN

**Review by puneet0191 — APPROVED** [review]
Please adjust the test title to be more descriptive, rest looks okay thank you.

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**